src/components/login-form.yml

```diff
@@ -3,11 +3,11 @@ loginForm:
     en: Please enter your password
     de: Bitte gib dein Kennwort an
   loginFailed:
-    en: Login failed. Please check your input.
-    de: Die Anmeldung ist fehlgeschlagen. Bitte überprüfe deine Eingabe.
+    en: The credentials you have entered are not correct.
+    de: Die Zugangsdaten sind nicht richtig.
```

Comment:
There is probably a better way to write this in German..

Reply:
Maybe we can also use "korrekt" in German: "Die Zugangsdaten sind nicht korrekt.", or to match the English translation completely: "Die Zugangsdaten, die du eingegeben hast, sind nicht korrekt." (might be too long, and it's kinda clear that they were entered by the user, though).

Reply:
I agree, "Die Zugangsdaten sind nicht korrekt." it is then.